solidity/contracts/BancorNetwork.sol

```diff
@@ -106,6 +107,20 @@ contract BancorNetwork is IBancorNetwork, TokenHolder, ContractRegistryClient {
         etherTokens[_token] = _register;
     }
 
+    /**
+      * @dev returns the conversion path between two tokens in the network
+      * note that this method is quite expensive in terms of gas and should generally be called off-chain
+      * 
+      * @param _sourceToken source token address
+      * @param _targetToken target token address
+      *
+      * @return conversion path between the two tokens
+    */
+    function conversionPath(IERC20Token _sourceToken, IERC20Token _targetToken) public view returns (address[]) {
+        IBancorNetworkPathFinder pathFinder = IBancorNetworkPathFinder(addressOf(CONVERSION_PATH_FINDER));

```

Comment:
@yudilevi : Can merge these two lines into one.

Reply:
I prefer not to have very long lines in the code and the cost is insignificant, so better keep it as is for readability.